tools/test/test262/template.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <excludeList>
     <!-- exclude each test file -->
+    <test id="built-ins/decodeURI/S15.1.3.1_A2.5_T1"><reason>SLOW</reason></test>
+    <test id="built-ins/decodeURI/S15.1.3.1_A2.5_T1"><reason>SLOW</reason></test>
+    <test id="built-ins/decodeURIComponent/S15.1.3.2_A2.5_T1"><reason>SLOW</reason></test>
+    <test id="built-ins/decodeURIComponent/S15.1.3.2_A2.5_T1"><reason>SLOW</reason></test>
     <test id="annexB/built-ins/RegExp/RegExp-leading-escape-BMP"><reason>SLOW</reason></test>
     <test id="annexB/built-ins/RegExp/RegExp-trailing-escape-BMP"><reason>SLOW</reason></test>
     <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-flags-u"><reason>SLOW</reason></test>
```

Comment:
Just a minor suggestion. Would you place these new lines in alphabetical order?
This is also applicable to `excludelist.orig.xml`.

Reply:
Sure. I reordered it.